File: hanzsgergo/week01/week-01-java-project/src/Loops/DrawChessTable.java
package Loops;

public class DrawChessTable {

  public static void main(String[] args) {
    // Crate a program that draws a chess table like this
    //
    // % % % %
    //  % % % %
    // % % % %
    //  % % % %
    // % % % %
    //  % % % %
    // % % % %
    //  % % % %
    //

    System.out.println("Chess table: ");
    //Rows
    for (int r = 0; r <= 8; r++) {

      //columns
      if (r % 2 != 0) {
        for (int c = 1; c <= 8; c++) {
            if (c % 2 == 0) {
                System.out.print("#");
            } else {
                System.out.print(" ");
            }
        }
      } else {
        for (int c = 1; c <= 8; c++) {
            if (c % 2 != 0) {
                System.out.print("#");
            } else {
                System.out.print(" ");
            }
        }
      }
      System.out.println();

    }
  }
}

File: Foundation/week_02-week_04/PythonBasicsProject/week_02/expressions_and_control_flow/loops/draw_chess_table.py
# Crate a program that draws a chess table like this
#
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
#

for y in range(8):
    for x in range(8):
        if y % 2 == 0:
            if x % 2 == 0:
                print(" # ", end="")
            else:
                print("   ", end="")
        else:
            if x % 2 != 0:
                print(" # ", end="")
            else:
                print("   ", end="")
    print()
